rotkehlchen/db/search_assets.py

```diff
@@ -0,0 +1,134 @@
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple
+
+from polyleven import levenshtein
+
+from rotkehlchen.assets.types import AssetType
+from rotkehlchen.constants.assets import A_ETH, A_ETH2
+from rotkehlchen.constants.resolver import ChainID
+from rotkehlchen.globaldb.handler import ALL_ASSETS_TABLES_QUERY, GlobalDBHandler
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+    from rotkehlchen.db.drivers.gevent import DBCursor
+
+
+def _search_only_nfts_levenstein(
```

Comment:
did you test these searches with nft heavy data accounts?

Reply:
No, will do it